### ISSUE TITLE
custom Class or Model error

### ISSUE THREAD

**Issue by wasiqbarazai** [yasarniyazoglu/YoloV5-and-DeepSort-Custom-Dataset#2]
Please can you help:
1) how this code works for custom Class/s (e.g., wheelBins) which is not in COCO?
2) It works for COCO classes and pre-trained YOLOv5s weights however, would be helpful if you can add steps how to USE CUSTOM-TRAINED YOLOv5s (comprising custom-CLASS) instead of original labels/weights?

In summary, Code/step does not work on newly trained weights (fine-tuned) for custom labels/class
